static/src/js/components/GroupList/GroupList.jsx

```diff
@@ -64,6 +69,13 @@ const GroupList = () => {
       userIds: userIds.length > 0 ? userIds : undefined
     }
   }
+
+  // If we are on the admin page, always use the 'CMR' provider
+  if (isAdmin) {
+    groupVariables.params.excludeTags = undefined
```

Comment:
I'm guessing you're attempting to delete this? Seems like there is a better way to do this.